cmd/iso_package/s3.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+type S3CredentialsConfig struct {
```

Comment:
Sorry again for commenting about naming, but the methods don't really fit the name of the struct. Why would credentials do an upload?